src/cascadia/TerminalSettingsModel/TerminalSettings.h

```diff
@@ -118,7 +118,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 
         INHERITABLE_SETTING(Model::TerminalSettings, hstring, ProfileName);
         INHERITABLE_SETTING(Model::TerminalSettings, bool, UseAcrylic, false);
-        INHERITABLE_SETTING(Model::TerminalSettings, double, TintOpacity, 0.5);
+        INHERITABLE_SETTING(Model::TerminalSettings, double, Opacity, UseAcrylic() ? 0.5 : 1.0);

```

Comment:
Interesting -- we do the defaulting at the TerminalSettings level instead of the Profile level

Reply:
Oh, that makes sense -- TSM feeds into the UI

Reply:
Here's a FUN question! How do we display this in the settings UI preview window? :D

Reply:
It kinda just looks... fainter? Since the SUI isn't transparent, that's what ends up showing behind the preview. I'll have to check out the branch again (tomorrow) to grab a SS.

Reply:
![image](https://user-images.githubusercontent.com/18356694/132762665-0e012c20-49d4-4192-ada2-8186b429ae70.png)

So yea, it's hard to tell, but the purple is a lot lighter because it's on top of white... if that makes sense